mula/scheduler/schedulers/boefje.py

```diff
@@ -558,10 +558,19 @@ def push_task(self, boefje: Plugin, ooi: OOI, caller: str = "") -> None:
             caller: The name of the function that called this function, used for logging.
 
         """
+
+        # TODO: see what types have a network attribute. How?
+        network_scope = "internet"
+        if ooi.object_type in ["IPAddressV4", "IPAddressV6"]:
```

Comment:
We could check if `ooi` has an attribute `network`, but this would rely on every OOI needing to name their network `network`. Is there a better solution to this?